stable/kubernetes-dashboard/README.md

```diff
@@ -58,7 +58,9 @@ The following table lists the configurable parameters of the kubernetes-dashboar
 | `service.internalPort`              | Dashboard internal port                                                                                                     | 443                                                                        |
 | `ingress.annotations`               | Specify ingress class                                                                                                       | `kubernetes.io/ingress.class: nginx`                                       |
 | `ingress.enabled`                   | Enable ingress controller resource                                                                                          | `false`                                                                    |
+| `ingress.gce`                       | Enables gce ingress controller specific configurations                                                                      | `false`                                                                    |
 | `ingress.path`                      | Path to match against incoming requests. Must begin with a '/'                                                              | `/`                                                                        |
+| `ingress.paths`                     | Paths to match against incoming requests. This is required to work with gce ingress `/` and `/*` required. Only when `ingress.gce` is true.                 | `[]`                                                                        |
```

Comment:
I was tempted to propose a breaking change but I decided to do it that way. If you are welling to do a breaking change I do not mind.

Reply:
If @kfox1111 agree I propose:
- `.Values.ingress.paths`, replacing `.Values.ingress.path`. 
- `.Values.ingress.path` is deprecated but still works (`if and .Values.ingress.path (not .Values.ingress.paths)` for example)

It may be worth looking at other charts to see how they handle this.

Reply:
If `.Values.ingress.paths` is defaulted to `[/]` then that condition will never work if someone defines `.Values.ingress.path` which is already defaulted to `/`.

Since this is a major revision, I do not think it is a huge change for somebody to switch. If using default path then the update to `1.0.0` is free. If not, it is only a matter of replacing `path: /` by `paths: [/]`